### PR TITLE
Backup/Restore controller

### PR DESCRIPTION
**Pull request by jacksonargo**
Provides a view for backups and restores.

### CODE REVIEW

**Comment by jacksonargo** [comment]
I think we can expand on this and add views and `click to restore` buttons! oh my the possibilities!

**Comment by 0az** [comment]
I would prefer to just keep it to a specific management script (or binary).

**Comment by jacksonargo** [comment]
Why is that? It's much simpler to just keep in the webapp. Far less code to write.

**Comment by jacksonargo** [comment]
Or rather, what are your concerns with having it in the webapp? 
What advantages does keeping it in separate binaries provide?

File: pkg/api/server.go [hunk 1]
```diff
@@ -41,7 +37,7 @@ func NewServer(ctx context.Context, config ServerConfig) *http.Server {
 
 	database := Database{
 		Distro: newBucket(ctx, config.DistroBucketName),
-		Parts:  parts.NewParts(config.RedisNamespace + ":parts"),
+		Parts:  parts.NewParts(config.RedisNamespace),
```
Comment (jacksonargo) [inline pkg/api/server.go:40]: The methods on Parts also prepend ":parts", so in redis it became `vvgo:parts:parts:id` lol

**Pull request merged**